src/app/client/src/app/modules/player-helper/components/player/player.component.ts

```diff
@@ -14,6 +14,11 @@ export class PlayerComponent implements OnInit, OnChanges {
   @Output() playerOnDestroyEvent = new EventEmitter<any>();
   @Output() sceneChangeEvent = new EventEmitter<any>();
   buildNumber: string;
+  contentRatingModal = false;
+  /**
+ * Dom element reference of contentRatingModal
+ */
```

Comment:
need to take input to show rating or not. In workspace player we should not show rating